main.cpp

```diff
@@ -0,0 +1,99 @@
+#include <iostream>
+#include <string>
+#include <fstream>
+
+using std::cout;
+using std::endl;
+using std::string;
+using std::cin;
+using std::getline;
+
+const string CONFIG_FILE_WRITE_PATH = "/Users/Jon/Desktop/C++repos/CPT-180-27-Assignment-2-Config-Generator/config.txt";
+
+int main(int argc, char *argv[]){
+// init
+    std::ofstream outputFile;
+    outputFile.open (CONFIG_FILE_WRITE_PATH);
+
+    string lastName;
+    cout << "Enter last name:" << endl;
+    getline(cin,lastName);
+    if(lastName == "")
```

Comment:
Around each of these inputs you should consider doing a loop so it continues to reprompt the user if they continue to not enter anything